grid/chat/members/maps/members.ts

```diff
@@ -21,12 +21,17 @@ export const getMembersTest = (
       });
 
       it('performs correctly', async () => {
+        const input: Record<string, number | string> = {
+          limit: 1,
+        };
+
+        if (typeof options?.workspace === 'string') {
+          input.workspace = options?.workspace;
```

Comment:
workspace in profile is defined as `workspace: string!`, which means if the field exists it must be a string. Here it can be `undefined` which is invalid. So, the `workspace` should be set only if it is a `string`.